src/twisted/trial/_dist/disttrial.py

```diff
@@ -287,7 +287,7 @@ class DistTrialRunner:
         ``False`` to run through the whole suite and report all of the results
         at the end.
 
-    @ivar _stream: stream which the reporter will use.
+    @ivar stream: stream which the reporter will use.
```

Comment:
Do we need it as public member?

I checked the diff, I don't see any usage, other than `self.stream`

Reply:
Unfortunately it is public on the de-facto "runner" interface defined by `TrialRunner`, which this class is trying to be compatible with.  It was a nice idea to make it private here but that violates the runner interface so it doesn't really work.  As soon as I made the protocol explicit with `_Runner` mypy began to complain that `DistTrialRunner` did not conform.